components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/scope/RoleBasedScopeIssuer.java

```diff
@@ -102,11 +102,16 @@ public class RoleBasedScopeIssuer extends AbstractRoleBasedScopeIssuer implement
     private static final String ISSUER_PREFIX = "default";
     OAuthServerConfiguration oAuthServerConfiguration = OAuthServerConfiguration.getInstance();
     private static final String REFRESH_TOKEN_GRANT_TYPE = "refresh_token";
+    private static final String ROLE_BASED_SCOPE_ISSUER_ENABLED =
+            "OAuth.GlobalScopeValidators.RoleBasedScopeIssuer.Enable";
 
     @Override
     public boolean validateScope(OAuthAuthzReqMessageContext oAuthAuthzReqMessageContext) throws
             IdentityOAuth2Exception {
 
+        if (!Boolean.parseBoolean(IdentityUtil.getProperty(ROLE_BASED_SCOPE_ISSUER_ENABLED))) {
```

Comment:
Shall we add method to read the config in https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

Reply:
Will be fixed with e941607122db5d7e7d6ffa95c12030f734c556cb